Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -202,17 +202,17 @@ function initialize(model) {
       });
       model._resourcesLoaded = true;
     })
-    .otherwise(function () {
-      ModelExperimentalUtility.getFailedLoadFunction(this, "model", resource);
-    });
+    .otherwise(
+      ModelExperimentalUtility.getFailedLoadFunction(model, "model", resource)
+    );
```

Comment:
This is a combination of two bugs:

* `getFailedLoadFunction` is creating the function that otherwise calls so it shouldn't be wrapped in a function
* `this` should be `model`